x-pack/plugin/searchable-snapshots/src/main/java/org/elasticsearch/xpack/searchablesnapshots/store/input/CachedBlobContainerIndexInput.java

```diff
@@ -272,7 +267,7 @@ protected MetadataCachingIndexInput doSlice(
             fileInfo,
             context,
             stats,
-            this.offset + sliceOffset,
+            sliceOffset,
```

Comment:
Was offset always 0 here (and in the other places of the PR)? Else, how does it work? :D

Reply:
Nah see here: https://github.com/elastic/elasticsearch/pull/98018/files#diff-e2f91bef43d1f6619cb62021a39d4493abca9ed41fef20cf97026a6c85983ed2R627 just deduplicated the addition logic because it's always `this.offset + ` in both implementations :)